Userland/Libraries/LibPDF/Document.h

```diff
@@ -70,7 +71,8 @@ struct OutlineDict final : public RefCounted<OutlineDict> {
     OutlineDict() = default;
 };
 
-class Document final : public RefCounted<Document> {
+class Document final : public RefCounted<Document>
+    , public AK::Weakable<Document> {
```

Comment:
Please format multiple inheritance with each class on its own line (& drop the AK namespace prefix)

```suggestion
class Document final
    : public RefCounted<Document>
    , public Weakable<Document> {
```

Reply:
Done